libtsuba/src/RDG.cpp

```diff
@@ -81,45 +81,29 @@ WriteProperties(
 }
 
 katana::Result<void>
-CommitRDG(
+ComputeManifest(
```

Comment:
Can you call this `SetTnxContextManifest`?  The point to me isn't so much that it is being computed as it is being held in memory as the "result" for this transaction.